lib/src/stream_subscription_transformer.dart

```diff
@@ -101,5 +100,5 @@ class _TransformedSubscription<T> implements StreamSubscription<T> {
   }
 
   Future/*<E>*/ asFuture/*<E>*/([/*=E*/ futureValue]) =>
-      _inner?.asFuture(futureValue) ?? new Completer().future;
+      _inner?.asFuture(futureValue) ?? new Completer/*<E>*/().future;
```

Comment:
Hmmm. Not sure, I know we have inference for `??`. Do you know what the error message was?